.travis.yml

```diff
@@ -6,5 +6,8 @@ install: npm install
 
 scripts: npm test
 
+before_script :
+  - psql -c 'CREATE DATABASE propertyTest;' -U postgres
```

Comment:
Keep it simple database variables. Postgres is very case-sensitive. Your test database can be named `property_test`. don't use camel case with Postgres names

Reply:
Already renamed  to postgres_test